### DISCUSSION TITLE
Issue/User Story count is wrong

### DISCUSSION THREAD

**Discussion by RobinWitt** [neuefische/questions#474] [Web] [answered]
Hi everyone, while writing user stories it just happened that GitHub decided to jump the number of my issues from `#9` to `#16` 
I didn't delete any Issues in between. Has anyone else had the same experience?
It's not really a problem, but still confusing and I am wondering why it happens.
Didn't find an answer online.

See a list of my issues [here](https://github.com/RobinWitt/capstone-project/issues)

**Comment by pixelass** [comment, marked as answer]
PRs are also issues: https://github.com/RobinWitt/capstone-project/pulls?q=is%3Apr+is%3Aclosed

**Reply by pixelass** [reply]
Also this shouldn't matter since it is just the ID of the issue.

**Reply by RobinWitt** [reply]
Thanks, I didn't see that. Good to know.
